test/integration/acceptance/acceptance_fixture.cpp

```diff
@@ -25,11 +34,24 @@ AcceptanceFixture::AcceptanceFixture()
           shared_model::crypto::DefaultCryptoAlgorithmType::generateKeypair()),
       kUserKeypair(
           shared_model::crypto::DefaultCryptoAlgorithmType::generateKeypair()),
+      checkEnoughSignatures([](auto &status) {
+        checkStatus<shared_model::interface::EnoughSignaturesCollectedResponse>(
```

Comment:
These are just comfy aliases to checkStatus itself.
It's much descriptive:
`itf.checkStatus(checkEnoughSignatures)`

rather than:
`itf.checkStatus(check_status<shared_model::interface::EnoughSignaturesCollectedResponse`

Reply:
OK, but is it possible to initialize the variables with the function directly, so that it is not required to wrap `checkStatus<T>` in a lambda? This will reduce code in this file at least.